Apps/ExampleDaemon/sdLitica.ExampleDaemonManagement/Program.cs

```diff
@@ -0,0 +1,134 @@
+﻿using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using sdLitica.Analytics;
+using sdLitica.Bootstrap.Extensions;
+using sdLitica.Events.Abstractions;
+using sdLitica.Events.Bus;
+using sdLitica.Events.Integration;
+using sdLitica.TimeSeries.Services;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Threading.Tasks;
+using Vibrant.InfluxDB.Client;
+using Vibrant.InfluxDB.Client.Rows;
+
+namespace sdLitica.ExampleDaemonManagement
+{
+    /// <summary>
+    /// Simple example of analytical module. 
+    /// </summary>
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            // configure dependency injection
+            var services = ConfigureServices();
+            var serviceProvider = services.BuildServiceProvider();
+
+            var _timeSeriesService = serviceProvider.GetRequiredService<ITimeSeriesService>();
+            var registry = serviceProvider.GetRequiredService<IEventRegistry>();
+
+            // register events
+            registry.Register<TimeSeriesAnalysisEvent>(Exchanges.TimeSeries);
+            registry.Register<DiagnosticsEvent>(Exchanges.Diagnostics);
```

Comment:
These statements are also used in Bootstrap.Events.ApplicationBuilderExtensions.SubscribeEvents(). 
They have to be in both places, because both subscription and publishing require an event to be present in EventRegistry object, which is also different in main app and daemon.